code/src/java/pcgen/system/PCGenSettings.java

```diff
@@ -104,6 +106,8 @@ private PCGenSettings()
 		setProperty(VENDOR_DATA_DIR, "@vendordata");
 		setProperty(HOMEBREW_DATA_DIR, "@homebrewdata");
 		setProperty(CUSTOM_DATA_DIR, "@data/customsources".replace('/', File.separatorChar));
+		OutputDB.registerBooleanPreference(
```

Comment:
This constructor is called from a static initialiser and thus this code is run before the properties are loaded. See pcgen.system.Main.loadProperties(boolean) for where the properties are loaded. The OutputDB would need to be initialised after that method is complete (or after its current processing).